Dockerfile

```diff
@@ -17,8 +18,8 @@ RUN echo "deb http://ftp.debian.org/debian stretch-backports main" >> /etc/apt/s
         sudo \
         vim \
         zip \
-    && DEBIAN_FRONTEND=noninteractive apt-get -t stretch-backports install -y \
-        python-certbot-apache \
+    # && DEBIAN_FRONTEND=noninteractive apt-get -t stretch-backports install -y \
+    #     python-certbot-apache \
```

Comment:
Do you want to keep this? I don't think commented out code should stay.

Reply:
Same as above comment. Will Fix.